tests/test_common_gpu.py

```diff
@@ -1083,6 +1085,77 @@ def test_8bit_dora_merging(self):
         assert torch.allclose(out_dora, out_unmerged, atol=atol, rtol=rtol)
         assert torch.allclose(out_dora, out_unloaded, atol=atol, rtol=rtol)
 
+    @require_torch_gpu
+    @pytest.mark.single_gpu_tests
+    def test_8bit_dora_cpu_offloading(self):
```

Comment:
Let's rename the test, as "8bit" is not being used. Also, let's use the name of the runtime argument in the test name, depending on what we will settle on.

Reply:
Good catch. Using ephemeral_transfers until a name is decided.